docs/pages/upgrading/client-tools-autoupdate.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: Teleport Client Tools Automatic Updates
+description: Explains how to use Teleport client tools (`tsh` and `tctl`) auto-updates.
+---
+
+This documentation explains how to keep Teleport client tools like `tsh` and `tctl` up-to-date.
+Updates can be automatic or self-managed, ensuring tools are secure, free from bugs,
+and compatible with your Teleport cluster.
+
+## Why Keep Client Tools Updated?
+
+- **Security**: Updates deliver patches for known vulnerabilities.
+- **Bug Fixes**: Resolved issues are pushed to endpoints.
+- **Compatibility**: Avoid manual understanding of [Teleport component compatibility rules](https://goteleport.com/docs/upgrading/overview/#component-compatibility).
+
+---
+
+## Automatic Updates
+
+### How It Works
+
+When you run `tsh login`, the tool will check if updates are enabled for your cluster.
```

Comment:
What does "the tool" refer to here? Is there an update tool that is separate from the client tools it keeps up to date?

Reply:
tool is a general term for tsh, tctl in this context, or potentially even tbot in the future

Reply:
👍 We should specify this in this paragraph. Otherwise, it is not possible for a reader to infer this.

Reply:
I've changed to `the tsh tool`, in scope of this example exactly tsh responsible for update